creations/silabs.md

```diff
@@ -0,0 +1,8 @@
+# silabs-creations
+Community Allocated Creation IDs for Silicon Labs boards
```

Comment:
These are official silabs ids right?

Reply:
The listed IDs are customized ones and asked by Silabs.

Our problem is that the microcontrollers and modules included in these development kits do not have built-in USB support. Therefore, we are using an additional MCU with Segger Jlink firmware for USB communication.

This same MCU and firmware are used for all of these boards, resulting in a lack of USB VIDs/PIDs for each board. As a result, we as Silabs are requiring these IDs, to support our own development, the linked CircuitPython porting for the xG24 devkits.

Reply:
Yup, this is good. I misunderstood the USB availability. Thanks!

Reply:
```suggestion
Creation IDs for Silicon Labs boards
```